### PR TITLE
Monitoring CloudFront

### PR DESCRIPTION
**Pull request by andreaswittig**
A monitoring jump start for CloudFront.

### CODE REVIEW

File: marbot-cloudfront.yml [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+  CloudFrontDistributionId:
+    Description: 'The CloudFront distribution ID that you want to monitor.'
+    Type: String
+  5xxErrorRateThreshold:
+    Description: 'The 5xx error rate threshold (set to -1 to disable).'
+    Type: Number
+    Default: 0.01
+    MinValue: -1
+  LambdaExecutionErrorThreshold:
+    Description: 'The Lambda@Edge execution error count threshold (set to >=0 to enable).'
```
Comment (michaelwittig) [inline marbot-cloudfront.yml:55]: change `set to >=0 to enable` to `set to -1 to disable`
A value of -2 would enable as well at the moment.

File: marbot-cloudfront.yml [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    Type: Number
+    Default: -1
+    MinValue: -1
+  LambdaValidationErrorThreshold:
+    Description: 'The Lambda@Edge validation error count threshold (set to >=0 to enable).'
```
Comment (michaelwittig) [inline marbot-cloudfront.yml:60]: change `set to >=0 to enable` to `set to -1 to disable`
A value of -2 would enable as well at the moment.

File: marbot-cloudfront.yml [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    Type: Number
+    Default: -1
+    MinValue: -1
+  CacheHitRateThreshold:
+    Description: 'The cache hit rate threshold (set treshold to enable, set to -1 to disable, requires detailed monitoring).'
```
Comment (michaelwittig) [inline marbot-cloudfront.yml:65]: remove `set treshold to enable, `

File: marbot-cloudfront.yml [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    Type: Number
+    Default: -1
+    MinValue: -1
+  OriginLatenyThreshold:
+    Description: 'The origin latency threshold in ms using the 99.5 percentile (set treshold to enable, set to -1 to disable, requires detailed monitoring).'
```
Comment (michaelwittig) [inline marbot-cloudfront.yml:70]: remove `set treshold to enable, `

**Comment by michaelwittig** [comment]
could you also add an entry to the README and list the new jump start?

**Comment by andreaswittig** [comment]
> could you also add an entry to the README and list the new jump start?

Done!

File: marbot-cloudfront.yml (hunk 4 above)
Comment (andreaswittig) [reply marbot-cloudfront.yml]: ✅

**Pull request merged**